external/js/cothority/README.md

```diff
@@ -91,3 +93,45 @@ will be provided thanks to the [buffer package](https://www.npmjs.com/package/bu
 As this is a [polyfill](https://remysharp.com/2010/10/08/what-is-a-polyfill), please
 check that what you need is implemented or you will need to use a different approach. Of
 course for NodeJS, you will always get a [Buffer](https://nodejs.org/api/buffer.html).
+
+### Use a developpment version from an external app
+
+**Note**: Using the official command `yarn link` doesn't seem to work. We leave
+the steps at the end for informational purpose, but we found out that using symlink
+is simpler, clearer, and actually works.
+
+Build and Link the local `@dedis/cothority` module to a local app:
+
+```bash
+(cothority/external/js/cothority) $ npm run build
+$ ln -s /<cothority root path>/cothority/external/js/cothority/dist /<local app root path>/node_modules/@dedis/cothority
```

Comment:
It seems easier to me to use `npm link @dedis/cothority` inside the local app root

Reply:
That doesn't work in my case. It links the `cothority` folder instead of the `cothority/dist` one.

Reply:
I mean you run `npm run link` in the cothority library (the script will link the dist folder) and you navigate to the external project and use `npm link @dedis/cothority`.